esm/interface/attr.js

```diff
@@ -40,9 +40,10 @@ export class Attr extends Node {
   }
 
   toString() {
-    const {name, [VALUE]: value} = this;
+    const {ownerDocument, name, [VALUE]: value} = this;
+    const doubleQuote = ownerDocument[MIME].unquotedJsonAttributes && /^\{(.[\s\S]?)+\}$/.test(value) ? '' : '"'
```

Comment:
what is `/^\{(.[\s\S]?)+\}$/` for? if it's about starting with `{` and ending with `}` I believe `/^\{[\s\S]*\}$/` is what you are after?

Reply:
Yep, I'll try that; should be more precise than my code there. It was late when I wrote that RegExp :)